<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Frame registration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/icons.qrc">
    <normaloff>:/images/AppLogo.png</normaloff>:/images/AppLogo.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7,2">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="tabs" name="widget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="stateIndicators" stretch="0,0,0">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="CPUWidget" name="cpuWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>CPU</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MemoryWidget" name="memoryWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>Memory</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="HddUsagePlot" name="hddWidget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar::item {
    spacing: 3px; /* spacing between menu bar items */
    padding: 1px 4px;
    background: transparent;
    border-radius: 2px;
}
QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #109910, stop: 1 #00ff00);
}
QMenuBar::item:pressed {
    background: #888888;
}</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="styleSheet">
     <string notr="true">QMenu {
    background-color: rgb(255,255,255); /* sets background of the menu */
    border-width: 1px;
    border-radius: 5px;
     border-color: grey;
    font: 11px;
}
QMenu::item{
 background-color: rgb(255, 255, 255);
 color: rgb(0, 0, 0);
}
QMenu::item:selected{
 background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #109910, stop: 1 #00ff00);
 color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionCzech"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="styleSheet">
     <string notr="true">QMenu {
    background-color: rgb(255,255,255); /* sets background of the menu */
    border-width: 1px;
    border-radius: 5px;
     border-color: grey;
    font: 11px;
}
QMenu::item{
 background-color: rgb(255, 255, 255);
 color: rgb(0, 0, 0);
}
QMenu::item:selected{
 background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #109910, stop: 1 #00ff00);
 color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionHDD_counter"/>
    <addaction name="actionCheck_version"/>
    <addaction name="actionCheck_for_update_automatically"/>
   </widget>
   <addaction name="menuLanguage"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="../images/icons.qrc">
     <normaloff>:/images/iconfinder_273_Ensign_Flag_Nation_kingdom_2634450.png</normaloff>:/images/iconfinder_273_Ensign_Flag_Nation_kingdom_2634450.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionCzech">
   <property name="icon">
    <iconset resource="../images/icons.qrc">
     <normaloff>:/images/iconfinder_71_Ensign_Flag_Nation_czech_2634499.png</normaloff>:/images/iconfinder_71_Ensign_Flag_Nation_czech_2634499.png</iconset>
   </property>
   <property name="text">
    <string>Czech</string>
   </property>
  </action>
  <action name="actionHDD_counter">
   <property name="text">
    <string>Add HDD counter name</string>
   </property>
  </action>
  <action name="actionEnable_auto_check">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable auto-check</string>
   </property>
  </action>
  <action name="actionCheck_version">
   <property name="text">
    <string>Check for update</string>
   </property>
  </action>
  <action name="actionCheck_for_update_automatically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Check for update automatically</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CPUWidget</class>
   <extends>QWidget</extends>
   <header location="global">power/cpuwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MemoryWidget</class>
   <extends>QWidget</extends>
   <header location="global">power/memorywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tabs</class>
   <extends>QWidget</extends>
   <header>main_program/tabs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HddUsagePlot</class>
   <extends>QWidget</extends>
   <header>power/hddusageplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/icons.qrc"/>
 </resources>
 <connections/>
</ui>
